Makefile

```diff
@@ -1,8 +1,8 @@
 
 # Use bash as the shell when executing a rule's recipe. For more details:
 # https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html
-# https://stackoverflow.com/questions/10376206/what-is-the-preferred-bash-shebang
-SHELL := /usr/bin/env bash
+SHELL := bash
+.SHELLFLAGS += -e
```

Comment:
Can you please add a comment?

Reply:
Let me remove this flag for now. From the [docs](https://www.gnu.org/software/make/manual/html_node/One-Shell.html):

> You can modify `.SHELLFLAGS` to add the `-e` option to the shell which will cause any failure anywhere in the command line to cause the shell to fail, but this could itself cause your recipe to behave differently. Ultimately you may need to harden your recipe lines to allow them to work with `.ONESHELL`.

So perhaps we should have a separte discussion about `.ONESHELL` and `.SHELLFLAGS` and exporting variables to the subshells.

Commit https://github.com/jenstroeger/python-package-template/pull/329/commits/1ebcf37b1cb24de5ecc7c210389b8d5c42354486.